book/history.md

```diff
@@ -120,8 +120,8 @@ This work also formed and inspired one of the key parts of Douglas Engelbart's
 [mother of all demos](https://en.wikipedia.org/wiki/The_Mother_of_All_Demos),
 perhaps the most influential technology demonstration in the history of
 computing (see Figure 3). That demo not only showcased the key concepts of the web, but also
-introduced the computer mouse and graphical user interface, both of which are of
```

Comment:
Removes double "of course" in two successive sentences. Should not affect reflow.